src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/BackedEnumValueResolver.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
```

Comment:
Shouldn't we actually trigger a deprecation?

```suggestion
namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;

trigger_deprecation('symfony/http-kernel', '7.3', 'The "%s" class is deprecated. Use "%s" from the "symfony/argument-resolver" component instead.', BackedEnumResolver::class, \Symfony\Component\ArgumentResolver\ValueResolver\BackedEnumValueResolver::class);
```
(same for others)

Reply:
`\Symfony\Component\ArgumentResolver\ValueResolver\BackedEnumValueResolver` does not exist in this PR. So what is the actual replacement ?

Reply:
The component provides the trait. The replacement is in HttpKernel https://github.com/symfony/symfony/pull/59794/files#diff-d9a7d766e7c8e96331d3f5ebd6bc528bb666c2331f3f86cbb750e0306e5b852a

Reply:
> Shouldn't we actually trigger a deprecation?

@mtarld BC layers are not finalized as mentioned in the PR body